organizations/organizations-router.js

```diff
@@ -0,0 +1,65 @@
+const express = require('express');
+const organizationsModel = require('./organizations-model');
+const campaignsModel = require('../campaigns/campaigns-model');
+const authenticate = require('../middleware/authenticate');
+
+const router = express.Router();
+
+router.get('/', async (req, res, next) => {
+  try {
+    const organizations = await organizationsModel.get();
+    res.status(200).json(organizations);
+  } catch (err) {
+    next(err);
+  }
+});
+
+router.get('/:id', async (req, res, next) => {
+  try {
+    const organization = await organizationsModel.getById(req.params.id);
+    res.status(200).json(organization);
+  } catch (err) {
+    next(err);
+  }
+});
```

Comment:
This endpoint also works, but it would be immensely helpful if this endpoint also returned a organization's campaigns.  That way, when a organization user logs in, the front end does not need to hit 1 endpoint to get the org name and then a 2nd endpoint to get the org's campaigns